e2e/playwright/info-drawer/.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "../../../.eslintrc.json",
+  "ignorePatterns": [
+    "!**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "parserOptions": {
+        "project": [
+          "e2e/playwright/list-views/tsconfig.e2e.json"
```

Comment:
shouldn't it be 'info-drawer' instead of 'list-views'?

Reply:
Correct, will be fixed in the next push